src/TBuffer.cpp

```diff
@@ -3178,26 +3184,26 @@ QString TBuffer::bufferToHtml(const bool showTimeStamp, const int row, const int
         // Pad out with spaces to the right so a partial first line lines up
     }
 
-    for (; pos < lastPos; ++pos) {
+    for (unsigned long cookedPos = static_cast<unsigned long>(pos); pos < lastPos; ++cookedPos, ++pos) {
```

Comment:
How about `for (auto cookedPos = static_cast<unsigned long>(pos);` so it doesn't repeat?

Reply:
:bulb: :+1: